Source/JavaScriptCore/debugger/Debugger.cpp

```diff
@@ -836,13 +836,13 @@ static inline JSGlobalObject* lexicalGlobalObjectForCallFrame(VM& vm, CallFrame*
     return callFrame->lexicalGlobalObject(vm);
 }
 
-void Debugger::updateCallFrame(JSGlobalObject* globalObject, CallFrame* callFrame, CallFrameUpdateAction action)
+void Debugger::updateCallFrame(JSGlobalObject* globalObject, CallFrame* callFrameMayBeNull, CallFrameUpdateAction action)
```

Comment:
It's kinda odd to have `MayBeNull` in the name.  I would've assumed that as a possibility since it's a pointer.

Reply:
OK, dropped.